docs/man_pages/publishing/publish-add.md

```diff
@@ -0,0 +1,18 @@
+publish add
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder publish add [<Name>] [<PublishUrl>]`
```

Comment:
I am slightly concerned that this publish command might cause confusion in users who have been using our other clients. They might be confused that we have a publish set of commands, appstore set of commands and appmanager set of commands. I am wondering whether the publish set of commands should be redesigned to nest the other commands?

In addition, it might be a good idea to rename the server connection set of commands to connection or something. And have a publish website command for the upload?